yarn-project/circuits.js/src/structs/kernel/new_contract_data.test.ts

```diff
@@ -0,0 +1,23 @@
+import { makeNewContractData } from '../../tests/factories.js';
+import { NewContractData } from './new_contract_data.js';
+
+describe('NewContractData', () => {
```

Comment:
Would it make sense for us to check any lengths here?

Reply:
Yes, addressed in https://github.com/AztecProtocol/aztec-packages/pull/4551/commits/9a12eaf436eb6b0c8239cee4ef913939a7181cae